src/typescript_proto_library.bzl

```diff
@@ -230,20 +304,54 @@ def _typescript_proto_library_impl(ctx):
         ],
     )
 
-typescript_proto_library = rule(
+def typescript_proto_library(name, proto):
```

Comment:
Let's rename this file to `typescript_proto_build` and move this macro to another file so it matches the other two macros.

Reply:
This is what initially this I had but it made reading the diff *very* difficult. Can we rename it as a separate PR after this one goes in?

Reply:
Sounds good to me.